plugins/gatsby-source-wagtail/test/mock_enterprise_page.json

```diff
@@ -5,7 +5,6 @@
     "title": "Example Enterprise",
     "slug": null,
     "last_published_at": "2019-07-16T20:57:48.578177Z",
-    "program_documents": null,
```

Comment:
Probably fine to remove this, but if the `pages` API keeps `program_documents` in its API response even for enterprise pages, might want to keep it in here just so it's a 1:1 mapping of the API response. If it's not in the API response for enterprise, yeah... let's remove this :)

Reply:
I am not including documents in the EnterprisePage model in designer for now (https://github.com/edx/portal-designer/pull/50). We can add it in if that becomes a requirement for the enterprise dashboard page later on.